examples/LoggingExtensions/Examples.LoggingExtensions.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\OpenTelemetry.Exporter.Console\OpenTelemetry.Exporter.Console.csproj" />
+    <ProjectReference Include="..\..\src\OpenTelemetry.Extensions.Serilog\OpenTelemetry.Extensions.Serilog.csproj" />
```

Comment:
nit: Use `$(RepoRoot)`.

Reply:
> nit: Use $(RepoRoot).

Updated

> Appears most Serilog sinks have Sink in the package name.

Right now the project is just a sink, but I don't know if it might contain other serilog artifacts in the future. For example say users are logging to flat files. They may want a serilog enricher that adds OTel context to those logs. We could also have that in this project. Basically my thinking was this project would just contain _stuff_ to help with different otel/serilog scenarios. We could of course have dedicated packages for each thing. More overhead to do that. Don't feel strongly either way so if you want it renamed just LMK and I'll update.

Reply:
> Don't feel strongly either way so if you want it renamed just LMK and I'll update.

Nah, I don't think `Sink` is needed allowing for additional Serilog things in the future.

The other thing I was pondering from a package naming perspective is whether we consider this an `Extension` or something else... Java for example considers their support for `log4j` "instrumentation" https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/log4j.

I think we have more time to consider these things.